### ISSUE TITLE
Can not modify language - Unknown culture

### ISSUE THREAD

**Issue by lOdwrot** [handsontable/react-handsontable#67]
I wanted to set language on column to de-DE and I've recived exception: 'Unknown culture : de-DE'. I' ve imported language file (min.js) from handsontable to component that uses react-handsontable and then I've tried to import this file on root html page through script tag without results. Is it a bug or I'm doing something wrong? Is there any support for not default culture?

**Comment by AMBudnik** [comment]
Hi @lOdwrot 

thanks for sharing
It looks like we have the same issue in Angular wrapper https://github.com/handsontable/angular-handsontable/issues/21

**Comment by lightsound** [comment]
Could you give me a sample code changing languages?
I read the following document and tried, but I could not get it work.
https://docs.handsontable.com/pro/4.0.0/tutorial-internationalization.html

I coded like this.
```
import "handsontable-pro/languages/ja-JP"

...

<HotTable 
  data={this.data}
  languages="ja-JP"
  contextMenu
/>
```

dependencies
```
"@handsontable-pro/react": "^1.0.0",
"handsontable-pro": "^4.0.0",
"next": "^6.0.3",
"react": "^16.4.0".
"react-dom": "^16.4.0"
```

**Comment by wojciechczerniak** [comment]
@lightsound Can you give us more info about your stack and build system? Tools, config, targeted module type, etc.

The default file under `import "handsontable-pro/languages/ja-JP"` is an UMD which is not always the right one for build tool configuration. There are scenarios where it's better to load ES module and register it by yourself:

    import jaJP from 'handsontable/es/i18n/languages/ja-JP';
    Handsontable.languages.registerLanguageDictionary('ja-JP', jaJP);

We're looking into this subject and your input will be helpful. Thanks.

**Comment by lightsound** [comment]
@wojciechczerniak Thanks. This is my environment. Is that enough?

stack: next, typescript
tools: webpack (=next)
module type: ES module

```javascript
// .babelrc
{
  "presets": ["next/babel", "@zeit/next-typescript/babel"],
  "plugins": [
    [
      "module-resolver",
      {
        "alias": {
          "~": "./src"
        }
      }
    ]
  ]
}
```

```javascript
// next.config.js
const withTypescript = require("@zeit/next-typescript");
const withCSS = require("@zeit/next-css");
module.exports = withCSS(withTypescript());
```

I tried your code, but I couldn't. In the first place, do we need to import Handsontable from `"handsontable-pro"` in react? Is it enough to just use HotTable from `"@handsontable-pro/react"`?

**Comment by wojciechczerniak** [comment]
Yes, you have to import both. Handsontable from `handsontable-pro` and `HotTable` from `@handsontable-pro/react`.

Thanks for the stack. It's very helpful.

**Comment by lightsound** [comment]
@wojciechczerniak Wow, good example.
I could change language with my stack.
Thanks very much!! 😆

**Comment by wojciechczerniak** [comment]
Great! 

I'm closing this ticket as original issue was fixed with Handsontable 4.0.
